docs/libraries/xml/guide.md

```diff
@@ -351,9 +356,9 @@ model Tag {
 
 ```xml
 <XmlPet>
-  <ItemsTag>
+  <XmlTag>
```

Comment:
here too, we have defined what the xml library does, the openapi3 implementation shouldn't affect this